TSOClient/tso.client/UI/Panels/EODs/UIRackEOD.cs

```diff
@@ -27,6 +27,44 @@ protected override void InitUI()
             btnPurchase.OnButtonClick += BtnPurchase_OnButtonClick;
         }
 
+        protected override void InitEOD()
+        {
+            base.InitEOD();
+            BinaryHandlers["rack_buy_error"] = PurchaseErrorHandler;
+        }
+
+        private void PurchaseErrorHandler(string name, byte[] type)
+        {
+            string title = GameFacade.Strings.GetString("264", "5"); // "Purchase Item"
+            String message = "";
+            // I already own this outfit - should do a custom error message but that means a new .cst entry...
```

Comment:
If they didn't have this error originally, I guess you could own multiple of the same? Not that I'd want to change it to that.

Reply:
Yeah, exactl. He only had a client side validation to enable the button. Circumventing that would have failed on outfits delivery, too.